02-intermediate-front-end/events-and-callbacks/readme.md

```diff
@@ -8,24 +8,67 @@
 - Pass a named function as a callback to another function
 - Pass an anonymous function as a callback to another function
 
```

Comment:
Could we add an opening framing here? Speaking to the bigger goal question. What we need events/callbacks for etc etc. 

---

We learned about objects and the DOM in JS last week. In order to do thing on the client side, we need programmatic access to the HTML/CSS with JS. Enter the DOM. This powerful tool allows us to interface with our HTML. Now that we have this capability we can create behaviors through JS that can act on HTML elements or be activated by HTML elements.


Or something like that...